README.md

```diff
@@ -131,6 +131,7 @@ This module exposes a single function `download` which takes the same arguments
   * **parquet** loads the urls and optional caption as a parquet
 * **url_col** the name of the url column for parquet and csv (default *url*)
 * **caption_col** the name of the caption column for parquet and csv (default *None*)
+* **bbox_col** the name of the bounding box column. Bounding boxes are assumed to have format ```[x_min, y_min, x_max, y_max]```, with all elements being floats in *[0,1]* (relative to the size of the image). If *None*, then no bounding box blurring is performed (default *None*)
```

Comment:
Is it better to have 0-1 values here rather than the absolute coordinate?

Don't most datasets use absolute values ?

Reply:
well I think this is probably reasonable